python/lesson1/tutorial.md

```diff
@@ -0,0 +1,5 @@
+#Introduction to Python
+
+There is a very good introduction article in [Google Developers Guide](https://developers.google.com/edu/python/introduction)
+
+Also there are resources for beginners on [the Python website](https://www.python.org/about/gettingstarted/). And you can always refer to [the documentation](https://docs.python.org/2/tutorial/introduction.html) that explains the basics.
```

Comment:
You can also find resources for beginners on [the Python website](https://www.python.org/about/gettingstarted/) and refer to [the Python documentation](https://docs.python.org/2/tutorial/introduction.html), where the language basics are explained.